docs/sources/predictlist/index.md

```diff
@@ -1,85 +1,87 @@
-# Predictlist
-If you would like to generate a list of predictions on already-trained Triage model with new data, you can use the 'Predictlist' module.
-
-# Predict Foward with Existed Model
-Use an existing model object to generate predictions on new data.
+# Retrain and Predict
+Use an existing model group to retrain a new model on all the data up to the current date and then predict forward into the future.
 
 ## Examples
 Both examples assume you have already run a Triage Experiment in the past, and know these two pieces of information:
-1. A `model_id` from a Triage model that you want to use to generate predictions
-2. An `as_of_date` to generate your predictions on.
+1. A `model_group_id` from a Triage model group that you want to use to retrain a model and generate prediction
+2. A `today` to generate your predictions on.
```

Comment:
I think we changed the argument to `prediction_date` rather than `today`, didn't we?